AppCenterDistribute/AppCenterDistribute/Internals/Ingestion/MSDistributeIngestion.m

```diff
@@ -15,21 +15,20 @@ @implementation MSDistributeIngestion
  * The API paths for latest release requests.
  */
 static NSString *const kMSLatestPrivateReleaseApiPathFormat = @"/sdk/apps/%@/releases/latest";
-static NSString *const kMSLatestPublicReleaseApiPathFormat = @"/public/sdk/apps/%@/distribution_groups/%@/releases/latest";
+static NSString *const kMSLatestPublicReleaseApiPathFormat = @"/public/sdk/apps/%@/releases/latest";
```

Comment:
we can do this change later in AB#75580 to avoid making the test app unusable until we have the change (can hardcode a group in the meantime).

Reply:
Does the new endpoint work?

Reply:
I made a stub for endpoint